src/django_components/__init__.py

```diff
@@ -1,52 +1,67 @@
-# flake8: noqa F401
 """Main package for Django Components."""
 
-import django
+import django as _django
 
 # Public API
-# isort: off
-from django_components.app_settings import ContextBehavior as ContextBehavior
-from django_components.autodiscover import (
-    autodiscover as autodiscover,
-    import_libraries as import_libraries,
-)
-from django_components.component import (
-    Component as Component,
-    ComponentView as ComponentView,
-)
-from django_components.component_registry import (
-    AlreadyRegistered as AlreadyRegistered,
-    ComponentRegistry as ComponentRegistry,
-    NotRegistered as NotRegistered,
-    RegistrySettings as RegistrySettings,
-    register as register,
-    registry as registry,
-)
-from django_components.components import DynamicComponent as DynamicComponent
-
 # NOTE: Middleware is exposed via django_components.middleware
-from django_components.dependencies import render_dependencies as render_dependencies
-from django_components.library import TagProtectedError as TagProtectedError
-from django_components.slots import (
-    SlotContent as SlotContent,
-    SlotFunc as SlotFunc,
+# NOTE: Some of the documentation is generated based on these exports
+from django_components.app_settings import ContextBehavior
+from django_components.autodiscovery import autodiscover, import_libraries
+from django_components.component import Component, ComponentVars, ComponentView
+from django_components.component_registry import (
+    AlreadyRegistered,
+    ComponentRegistry,
+    NotRegistered,
+    RegistrySettings,
+    register,
+    registry,
 )
+from django_components.components import DynamicComponent
+from django_components.dependencies import render_dependencies
+from django_components.library import TagProtectedError
+from django_components.slots import SlotContent, SlotFunc
 from django_components.tag_formatter import (
-    ComponentFormatter as ComponentFormatter,
-    ShorthandComponentFormatter as ShorthandComponentFormatter,
-    TagFormatterABC as TagFormatterABC,
-    TagResult as TagResult,
-    component_formatter as component_formatter,
-    component_shorthand_formatter as component_shorthand_formatter,
+    ComponentFormatter,
+    ShorthandComponentFormatter,
+    TagFormatterABC,
+    TagResult,
+    component_formatter,
+    component_shorthand_formatter,
 )
-from django_components.template import cached_template as cached_template
+from django_components.template import cached_template
 import django_components.types as types
-from django_components.util.types import (
-    EmptyTuple as EmptyTuple,
-    EmptyDict as EmptyDict,
-)
+from django_components.util.types import EmptyTuple, EmptyDict
 
-# isort: on
 
-if django.VERSION < (3, 2):
+if _django.VERSION < (3, 2):
```

Comment:
Also, I wonder if we can remove this logic, since Django 3.2 isn't supported since Apr 2024?

Reply:
Definitely! Anything not here can be removed: https://github.com/EmilStenstrom/django-components?tab=readme-ov-file#compatibility

Reply:
Alright, updated!